pkgs/development/python-modules/tensorflow/default.nix

```diff
@@ -149,7 +149,7 @@ let
   ];
 
   cudatoolkitDevMerged = symlinkJoin {
-    name = "cuda-${cudaPackages.cudaVersion}-dev-merged";
+    name = "cuda-${cudaPackages.cudaMajorMinorVersion}-dev-merged";
```

Comment:
I understand the `cudaOlder` part, but what is the motivation for stripping the version strings?

Reply:
Is `cudaVersion` guaranteed to be major.minor, or is that just a coincidence